keras/engine/saving.py

```diff
@@ -24,7 +24,7 @@
     h5py = None
 
 
-def _serialize_model(model, f, include_optimizer=True):
+def _serialize_model(model, h5dict_instance, include_optimizer=True):
```

Comment:
An `_instance` suffix is not good style.

Reply:
which one is better?

Reply:
I think `h5dict` is enough for a reader to understand what is going on.

Reply:
Sounds good.